src/tezos_rpc/fetch_block_operations.ml

```diff
@@ -0,0 +1,176 @@
+open Helpers
+open Crypto
+open Tezos
+open Http
+
+type parameters = {
+  entrypoint : string;
+  value : Michelson.t;
+}
+type internal_operation =
+  | Internal_transaction     of {
+      sender : Address.t;
+      destination : Address.t;
+      parameters : parameters option;
+    }
+  | Internal_non_transaction
+
+type operation_status =
```

Comment:
[Format question] Why is there a blank line between the definition of `type internal_operation` and this one, but not between this one and the following `type operation`? 🤔

Reply:
Mostly this is a grouping cue, `internal_operation` and `parameters` make a single entity an `internal_operation`, where `operation_status`, `operation`(bad name) and `block_operation` makes an `block_operation`.